tui/profiles.sh

```diff
@@ -25,6 +28,11 @@ for method in "${available_profiles[@]}"; do
 done
 
 PROFILE=$(whiptail "${whiptail_args[@]}" 3>&1 1>&2 2>&3)
+# Logic to go back to  screen
+if [ "$PROFILE" == $BACK_BUTTON ]; then
```

Comment:
Missing the `"` around `$BACK_BUTTON`.

Reply:
Thanks for feedback
PS: Sorry for the poor quality of code its my first time working on a big project